README.md

```
@@ -39,6 +39,11 @@ Download and install [emscripten]:
 task emsdk
 ```
 
+Download the required source dependencies:
+```shell
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)